### PR TITLE
Add getting started guide

### PR DESCRIPTION
**Pull request by Horicabu**
Related to https://github.com/ZigRazor/CXXGraph/issues/407.

Adds additional documentation to the Docusaurus site that supplements the existing 'How to Use' document and adds in a getting started guide. Revised PR.

### CODE REVIEW

File: docusaurus/docs/getting-started/how-to-use.md [hunk 1]
```diff
@@ -4,4 +4,13 @@ title: 'How to Use CXXGraph'
...
+
+To use CXXGraph in your project, simply put the header file `CXXGraph.hpp` where you need it, and include the following in your header:
+
+```
+include "CXXGraph.hpp"
+
+using namespace CXXGRAPH;
+```
+
+It's that easy!
```
Comment (nolankramer) [inline docusaurus/docs/getting-started/how-to-use.md:16]: Sadly, not yet! We haven't created a deployment/combined header that can be simply included into a project. I think @ZigRazor is tracking some work to add that.

File: docusaurus/docs/getting-started/how-to-use.md [hunk 2]
```diff
@@ -4,4 +4,13 @@ title: 'How to Use CXXGraph'
...
 ### How do I put CXXGraph on my project?
-To use the library simply **put the header file where you need it.** It’s that easy!
\ No newline at end of file
+
+To use CXXGraph in your project, simply put the header file `CXXGraph.hpp` where you need it, and include the following in your header:
+
+```
+include "CXXGraph.hpp"
+
+using namespace CXXGRAPH;
```
Comment (nolankramer) [inline docusaurus/docs/getting-started/how-to-use.md:13]: `using namespace CXXGRAPH;` -> `using namespace CXXGraph;`

File: docusaurus/docs/getting-started/how-to-make-a-graph.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+### Adding Nodes and Edges
+
+To begin using the graph, we require nodes to create edges between. Nodes are created individually.
+
+Here we create two nodes...
+
+```
+CXXGRAPH::Node<int> node1("1", 1);
+CXXGRAPH::Node<int> node2("2", 2);
```
Comment (nolankramer) [inline docusaurus/docs/getting-started/how-to-make-a-graph.md:31]:
```
CXXGRAPH::Node<int> node1("1", 1);
CXXGRAPH::Node<int> node2("2", 2);
```

Should be

```
CXXGraph::Node<int> node1("1", 1);
CXXGraph::Node<int> node2("2", 2);
```

All the references below should change `CXXGRAPH` -> `CXXGraph`

File: docusaurus/docs/getting-started/how-to-use.md (hunk 1 above)
Comment (ZigRazor) [reply docusaurus/docs/getting-started/how-to-use.md:16]: This is true after the installation, @nolankramer  can you open an issue to track this missing, or almost misconfigured, installation phase in cmake? 
Thank you in advance, I think in any case that this pull request can be merged, you agree?

File: docusaurus/docs/getting-started/how-to-make-a-graph.md (hunk 3 above)
Comment (ZigRazor) [reply docusaurus/docs/getting-started/how-to-make-a-graph.md:31]: Yes this is true, @Horicabu  can you correct these?

File: docusaurus/docs/getting-started/how-to-use.md (hunk 1 above)
Comment (ZigRazor) [reply docusaurus/docs/getting-started/how-to-use.md:16]: I have opened the new Issue

**Comment by Horicabu** [comment]
I can gladly correct the code samples!

File: docusaurus/docs/getting-started/how-to-use.md (hunk 1 above)
Comment (nolankramer) [reply docusaurus/docs/getting-started/how-to-use.md]: If we update the instructions here to specify using CPM (which I use in my own project), this should be mergeable.

For now, using CPM looks like this:
```
CPMAddPackage("gh:ZigRazor/CXXGraph")
if (CXXGraph_ADDED)
    add_library(CXXGraph INTERFACE IMPORTED GLOBAL)
    target_include_directories(CXXGraph INTERFACE "${CXXGraph_SOURCE_DIR}/include")
endif()
```

Comment (Horicabu) [reply docusaurus/docs/getting-started/how-to-use.md]: If Zigrazor wants this in the how to use doc I'll gladly add a section.

**Review by ZigRazor — APPROVED** [review]

**Comment by ZigRazor** [comment]
Good Job I merge it!

**Pull request merged**